### DISCUSSION TITLE
Adding VCV prototype module

### DISCUSSION THREAD

**Discussion by dinegredin** [DISTRHO/Cardinal#255] [Ideas]
Is there a chance that VCV prototype could be added in the future ?
It would be really fun to be able to mess with pd patches and more inside Cardinal.
Thanks for your great work

**Comment by dromer** [comment]
I think unlikely.

The VCV Prototype module depends on a lot of external tooling and interpreters. This would make the Cardinal builds a lot more complex (and lengthy) and make cross-platform and cross-architecture support really difficult as well. Ultimately the benefits are minimal and the match with the usecase of Cardinal isn't all to good either.

Just my 2ct

**Reply by dinegredin** [reply]
Ok, thanks for your answer anyway.
